front/locale.php

```diff
@@ -44,28 +44,24 @@
 use function Safe\json_encode;
 use function Safe\preg_match;
 use function Safe\preg_replace;
-use function Safe\session_write_close;
 
 global $CFG_GLPI, $TRANSLATE;
 
-session_write_close(); // Unlocks session to permit concurrent calls
```

Comment:
Is there a reason this was deleted? I don't see anything that write to the session in the changes.

Reply:
Alright, looks good then 👍